client/jetpack-connect/style.scss

```diff
@@ -115,7 +135,6 @@
 .jetpack-connect__main.is-wide {
 	max-width: 100%;
 	text-align: center;
-	margin-bottom: 24px;
```

Comment:
Can we make sure this doesn't affect any other steps in an unexpected way?

Reply:
I've tested all the flows I could think of, and noticed no problems.

Reply:
Yeah, I've given this a bit more testing too, seems like it only removes some unnecesary spacing at the bottom of the page. I think we're good 👍